resources.html

```diff
@@ -42,223 +19,40 @@
 
 <h2>Other Resources</h2>
 
-<p>Omnigraffle&reg; and Word&reg; files used for the creation of the screenshots used in the User Help website and for the PDF versions for printing.</p>
-
-<p>These source files can be downloaded from the Resources directory in OME Downloads:</p>
+<p>Omnigraffle&reg; files used for the creation of the screenshots used in the User Help website and other resources such as Word&reg; files can be downloaded from the Help resources directory on the OME Downloads site:</p>
```

Comment:
👍 Just didn't want to delete them and then have you say you wanted to keep more of the content. I'll get rid of them now.